Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.yml

```diff
@@ -84,9 +84,9 @@ configuration:
   required: true
   type: 4
 - defaultvalue: connectApi
-  display: Sub-Feeds
+  display: Services
   hidden: false
-  name: sub_feeds
+  name: x_services
```

Comment:
?

Reply:
Thought maybe `services` is a keyword, but it didnt help